Task 1

SeredaM98 opened the pull request.

SeredaM98 committed:
Add files via upload

mblagov commented on src/main/java/ru/spbu/apcyb/svp/tasks/Task1.java:
```diff
@@ -1,11 +1,137 @@
+      throw new RuntimeException("Неверный ввод суммы размена.");
+    }
+    String str = in.nextLine();
+    long[] notes = get_notes(str);
```
Используйте код стайл  java. Camel case  для переменных с маленькой буквы

mblagov commented on src/main/java/ru/spbu/apcyb/svp/tasks/Task1.java:
```diff
@@ -1,11 +1,137 @@
+      }
+    }
+    Arrays.sort(notes);
+    ArrayList<Long> newnotes = new ArrayList<>();
```
Что будет, если номиналы дублируются?

SeredaM98 committed:
Исправления

Переименовал переменные в стиле Camel case.
Использовал Set для предотвращения дублирования купюр.
